var/spack/repos/builtin/packages/libxau/package.py

```diff
@@ -17,6 +17,9 @@ class Libxau(AutotoolsPackage, XorgPackage):
 
     maintainers("wdconinc")
 
+    build_system("autotools", conditional("meson", when="@1.0.12:"), default="autotools")
```

Comment:
Can we keep the default consistent with the latest/preferred version?

Reply:
This is the only Xorg library that supports meson and autotools continues to be supported. I don't think it makes sense (yet) to pull in meson and its deps by default when someone just needs libx11, only because of libxau.

Reply:
Maybe in the future this will change. Just in the past weekend a large number of PRs were opened to add meson to the other X libraries: https://gitlab.freedesktop.org/search?group_id=2211&scope=merge_requests&search=meson